src/features/projectsV2/ProjectDetails/components/SinglePlantInfo.tsx

```diff
@@ -0,0 +1,48 @@
+import PlantInfoCard from './microComponents/PlantInfoCard';
+import { formatHid } from '../../../../utils/projectV2';
+import styles from '../styles/PlantLocationInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import {
+  PlantLocation,
+  SamplePlantLocation,
+} from '../../../common/types/plantLocation';
+import getImageUrl from '../../../../utils/getImageURL';
+import TreeMapperBrand from './microComponents/TreeMapperBrand';
+
+interface Props {
+  plantData: SamplePlantLocation | PlantLocation | null;
```

Comment:
Update types to make more specific. This does not allow multiple tree registrations.